llvm/lib/Target/NVPTX/NVPTXIntrinsics.td

```diff
@@ -7194,7 +7197,27 @@ class WMMA_REGINFO<WMMA_REGS r, string op>
 
     !and(!eq(op,"ldmatrix"),
          !eq(ptx_elt_type,"b16"),
-         !eq(geom, "m8n8")) : [hasSM<75>, hasPTX<65>]);
+         !eq(geom, "m8n8")) : [hasSM<75>, hasPTX<65>],
+
+    !and(!eq(op,"ldmatrix"),
+         !eq(ptx_elt_type,"b8"),
+         !eq(geom, "m16n16")) : [hasSM<100>, hasAAFeatures, hasPTX<86>],
```

Comment:
Nit: We already have `hasSM100a`:
https://github.com/llvm/llvm-project/blob/3c2ba68915b268fd3b7d39bf62e19199b2cb8995/llvm/lib/Target/NVPTX/NVPTXInstrInfo.td#L175-L180

Future cleanup opportunity, in a separate patch:

`hasAAFeatures` still rubs me the wrong way.  When I see `AA` my brain jumps straight to "what does it have to do with the Alias Analysys?". Also, it looks like 'accelerated' SM variants are here to stay.

Perhaps we can generalize `hasSM` to accept an optional "accelerated" argument, or just create `hasSMA<xx>`.

Reply:
Thanks to @durga4github's PR (https://github.com/llvm/llvm-project/pull/127990), I have updated my change to use `hasArchAccel` predicated instead of `hasAAFeatures`